data/mods/CRT_EXPANSION/items/crt_toolarmor.json

```diff
@@ -220,10 +217,34 @@
     "use_action": "HOTPLATE"
   },
   {
-    "id": "bandana_wet",
+    "id": "packaged_bandana_wet",
```

Comment:
You'd have to obsolete the old ID and migrate to this one, or it breaks.

Reply:
Do I remove it?

Reply:
2. Create a migration.json file inside the items folder, add this to it:
```
{
    "id": "bandana_wet",
    "type": "MIGRATION",
    "replace": "packaged_bandana_wet"
  }
```
3. If there's an associated recipe already, add an obsolete.json to *recipes* folder and add this:
```
{
    "type": "recipe",
    "result": "bandana_wet",
    "obsolete": true
  }
```

Reply:
Oh ok, nice!